study-builder/fdahpStudyDesigner/src/main/java/com/fdahpstudydesigner/bo/ConsentBo.java

```diff
@@ -48,6 +48,7 @@
 
   private static final long serialVersionUID = 5564057544960167010L;
 
+  @Deprecated
```

Comment:
add google 2020-2021 to copyright like other files

Reply:
Fixed review comment.